_locales/_untranslated_en.json

```diff
@@ -55,5 +55,26 @@
   },
   "ask_send_file_desktop":{
     "message": "Send %s to %s?"
+  },
+  "confirm_leave_group": {
+    "message": "Leave"
+  },
+  "confirm_delete_chat": {
+    "message": "Delete"
+  },
+  "confirm_block_contact": {
+    "message": "Block"
+  },
+  "confirm_unblock_contact": {
```

Comment:
please just name the string "unblock" so it can be used more widely. we want to have special sorted translations only if differencing is really required - and this is usually not forseeable

Reply:
the is way a longer string and it will be split in two lines, you still want that?

Reply:
i think, in general, a button with 2 words should work on desktop. i would leave it, we can restyle the things later on. this functionality is not used so frequently, so it is fine this way for now.